sdk/identity/azure-identity/src/main/java/com/azure/identity/AadCredentialBuilderBase.java

```diff
@@ -77,4 +80,31 @@ public T executorService(ExecutorService executorService) {
         this.identityClientOptions.setExecutorService(executorService);
         return (T) this;
     }
+
+    /**
+     * For multi-tenant applications, specifies additional tenants for which the credential may acquire tokens.
+     * Add the wildcard value "*" to allow the credential to acquire tokens for any tenant on which the application is installed.
+     *
+     * @param additionallyAllowedTenants the additionally allowed Tenants.
```

Comment:
This param line is the only place you upper-case the T for Tenants

Reply:
fixed.